src/selectors/entities/channels.js

```diff
@@ -363,11 +363,20 @@ export const getMyChannels: (GlobalState) => Array<Channel> = createSelector(
     }
 );
 
-export const getOtherChannels: (GlobalState) => Array<Channel> = createSelector(
+export const getOtherChannels: (GlobalState, ?boolean) => Array<Channel> = createSelector(
+    getChannelsInCurrentTeam,
+    getMyChannelMemberships,
+    (state: GlobalState, archived: ?boolean = true) => archived,
```

Comment:
does it default to true? I would think this defaults to `false` as they were not included originally

Reply:
Yep. Defaults to true. It did return archived channels before